### PR TITLE
[ci] we know, flare is broken

### PR DESCRIPTION
**Pull request by degemer**
No need to keep the CI broken.
It's being worked on.

### CODE REVIEW

**Comment by yannmh** [comment]
👍 
Can you prepare a PR with a revert please ?

File: tests/core/test_flare.py [hunk 1]
```diff
@@ -168,21 +168,6 @@ def test_upload_with_case_proxy(self, mock_config, mock_tempdir, mock_stfrtime,
         assert kwargs['proxies']['https'] == 'http://proxy_user:proxy_pass@proxy.host.com:3128'
         assert not kwargs['verify']
 
-    @attr(requires='core_integration')
```
Comment (truthbk) [inline tests/core/test_flare.py]: why don't we just add a `skip` attribute for when these things happen?

File: tests/core/test_flare.py [hunk 2]
```diff
@@ -1,11 +1,12 @@
 # stdlib
-import os.path
+import os
 import unittest
 import re
 
 # 3p
 import mock
 from nose.plugins.attrib import attr
+from unittest.case import SkipTest
```
Comment (yannmh) [inline tests/core/test_flare.py]: #stdlib ?

Comment (degemer) [inline tests/core/test_flare.py]: 😵

**Pull request merged**